### PR TITLE
docs: move "Error handling" to "Error codes" in "Reference guides"

### PR DESCRIPTION
**Pull request by bajtos**
This is a follow-up to #5718, see also #5549.

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- Affected artifact templates in `packages/cli` were updated
- Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/reference/error-codes.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
 ---
 lang: en
-title: Error Handling
+title: Error Codes
 keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI, Error Handling
 sidebar: lb4_sidebar
-permalink: /doc/en/lb4/Error-handling.html
+permalink: /doc/en/lb4/reference-error-codes.html
```
Comment (raymondfeng) [inline docs/site/reference/error-codes.md:6]: `Refence-error-codes.html`

File: docs/site/sidebars/lb4_sidebar.yml [hunk 2]
```diff
@@ -563,6 +559,10 @@ children:
     url: apidocs.index.html
     output: 'web, pdf'
 
+  - title: 'Error codes'
+    url: reference-error-codes.html
```
Comment (raymondfeng) [inline docs/site/sidebars/lb4_sidebar.yml:563]: `Reference-error-...`

File: docs/site/reference/error-codes.md (hunk 1 above)
Comment (bajtos) [reply docs/site/reference/error-codes.md:6]: See https://github.com/strongloop/loopback-next/pull/5775#discussion_r443347719 and https://github.com/strongloop/loopback-next/issues/5798

**Comment by bajtos** [comment]
Based on discussion with @raymondfeng, we will stick with Sentence-style.html to land my PRs and then normalize URL slugs afterwards, as this part may take a bit longer to reach consensus.

**Pull request merged**